### PR TITLE
feat(metrics): bring back some mem table memory metrics

### PR DESCRIPTION
**Pull request by wcy-fdu**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?
Previously we remove the mem table metrics due to the refresh is too fast and cannot be monitored, but it can be useful in some case such as temporal filter can emit a large number of messages to downstream in one barrier, and these metrics is useful to test/monitor mem table spill.
 let's bring it back.

## Checklist

- [ ] I have written necessary rustdoc comments
- [ ] I have added necessary unit tests and integration tests
- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [ ] My PR contains breaking changes. (If it deprecates some features, please create a tracking issue to remove them in the future).
- [ ] All checks passed in `./risedev check` (or alias, `./risedev c`)
- [ ] My PR changes performance-critical code. (Please run macro/micro-benchmarks and show the results.)

- [ ] My PR contains critical fixes that are necessary to be merged into the latest release. (Please check out the [details](https://github.com/risingwavelabs/risingwave/blob/main/CONTRIBUTING.md))

## Documentation

- [ ] My PR needs documentation updates. (Please use the **Release note** section below to summarize the impact on users)

## Release note

If this PR includes changes that directly affect users or other significant modifications relevant to the community, kindly draft a release note to provide a concise summary of these changes. Please prioritize highlighting the impact these changes will have on users.

</details>

### CODE REVIEW

File: grafana/risingwave-dev-dashboard.dashboard.py [hunk 1]
```diff
@@ -1591,6 +1591,38 @@ def section_batch(outer_panels):
                         ),
                     ],
                 ),
+
+                panels.timeseries_bytes(
+            "Mem Table Size",
```
Comment (fuyufjh) [inline grafana/risingwave-dev-dashboard.dashboard.py:1596]: Can you please help to revert back the metric `Mem Table Size (Max)` in L2215:

```
-                    "Mem Table Size (Max)",
-                    "Write Btach Size",
```

And add a `avg` for it as well. Thanks a lot!

File: src/storage/src/monitor/hummock_state_store_metrics.rs [hunk 2]
```diff
@@ -341,6 +345,23 @@ impl HummockStateStoreMetrics {
             metric_level,
         );
 
+        // todo(wcy-fdu): may replace instance_id with actor_id.
```
Comment (fuyufjh) [inline src/storage/src/monitor/hummock_state_store_metrics.rs:348]: May use `metrics_level` if you want. (But current state LGTM)

**Review by fuyufjh — APPROVED** [review]

**Review by fuyufjh — APPROVED** [review]
LGTM

File: src/storage/src/monitor/hummock_state_store_metrics.rs (hunk 2 above)
Comment (wcy-fdu) [reply src/storage/src/monitor/hummock_state_store_metrics.rs:348]: We may remove this metrics soon after mem table can spill.

**Pull request merged**